File: command/java/receivers/TextEditor.java
package command.java.receivers;


public class TextEditor {
    private String content = "";

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public void copy(String text) {
        System.out.println("Copying: " + text);
    }

    public void cut(String text) {
        System.out.println("Cutting: " + text);
        content = content.replace(text, "");
    }

    public void paste(String text) {
        System.out.println("Pasting: " + text);
        content += text;
    }
}

File: command/python/receivers/text_editor.py

class TextEditor:
    def __init__(self):
        self._content = ""

    def get_content(self):
        return self._content

    def set_content(self, content):
        self._content = content

    def copy(self, text):
        print(f"Copying: {text}")

    def cut(self, text):
        print(f"Cutting: {text}")
        self._content = self._content.replace(text, '')

    def paste(self, text):
        print(f"Pasting: {text}")
        self._content += text

